test/load_path_cache/core_ext/kernel_require_test.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require('test_helper')
+
+module Bootsnap
+  module LoadPathCache
+    class CacheTest < MiniTest::Test
```

Comment:
```suggestion
    class KernelLoadTest < MiniTest::Test
```
The file should also probably be `kernel_load_test.rb`

Reply:
copypaste is bad, fixed.